airbyte-integrations/connectors/source-gcs/acceptance-test-config.yml

```diff
@@ -4,24 +4,34 @@ connector_image: airbyte/source-gcs:dev
 acceptance_tests:
   spec:
     tests:
-      - spec_path: "source_gcs/spec.yaml"
+      - spec_path: integration_tests/spec.json
+        backward_compatibility_tests_config:
+          disable_for_version: 0.2.0
   connection:
     tests:
       - config_path: "secrets/config.json"
-        status: "succeed"
+        status: succeed
       - config_path: "integration_tests/invalid_config.json"
-        status: "failed"
+        status: exception
   discovery:
     tests:
       - config_path: "secrets/config.json"
+        timeout_seconds: 2400
   basic_read:
     tests:
       - config_path: "secrets/config.json"
         configured_catalog_path: "integration_tests/configured_catalog.json"
-        empty_streams: []
+        expect_trace_message_on_failure: false
```

Comment:
Could you please add a comment explaining why there are no expected records?

Reply:
because this is alpha connector and no need add high strictness in acceptance tests, so expected records no tested here.